osrf_testing_tools_cpp/src/memory_tools/vendor/bombela/backward-cpp/BackwardConfig.cmake

```diff
@@ -50,7 +50,7 @@ endif()
 ###############################################################################
 # CONFIGS
 ###############################################################################
-if (${STACK_DETAILS_AUTO_DETECT})
+if (${STACK_DETAILS_AUTO_DETECT} AND NOT WIN32)
```

Comment:
Should we document this change somehow for posterity? I.e. if we update Backward in a similar way in the future, is it possible we will lose this change accidentally?

Reply:
Yes, but I don't know where is the best place to do that. You will see the commit on top when you try to do a subtree update. I planned on opening a pull request to them, but I'm trying to prioritize other stuff atm.

Any idea on where to add a note that wouldn't get lost?

Reply:
I don't know either.

> You will see the commit on top when you try to do a subtree update.

That might be good enough.